HTTPServerProject.Tests/HTTPServerProxyTests/CheckPathTests.cs

```diff
@@ -0,0 +1,16 @@
+using HTTPServerProxy.Check.Paths;
+
+namespace HTTPServerProxyTests.Check.Paths;
+
+public class HttpPathTests
+{
+    [Theory]
+    [InlineData("todo", true)]
+    [InlineData("tod", false)]
+    public void GetTodoPathTrueTest(string path, bool expected)
```

Comment:
The name of the test suggests you're testing for a `true` scenario, but I think this test would run twice; (1) testing a scenario where you can find the path `todo` (e.g. `true`) and (2) testing a scenario where you cannot find the path `todo` (e.g. `false`). Maybe the test name should be changed to `ValidateTodoPathTest`? What are your thoughts?

Reply:
That's a good point, I will update for that!